common/pb/serverinfo_room.proto

```diff
@@ -6,10 +6,11 @@ message ServerInfo_Room {
 	optional sint32 room_id = 1;
 	optional string name = 2;
 	optional string description = 3;
-	optional uint32 game_count = 4;
-	optional uint32 player_count = 5;
-	optional bool auto_join = 6;
-	repeated ServerInfo_Game game_list = 7;
-	repeated ServerInfo_User user_list = 8;
-	repeated ServerInfo_GameType gametype_list = 9;
+	optional string permissionlevel = 4;
```

Comment:
From what I can tell it shouldn't.  All the protocol parts are referenced/set by there descriptive name and not number.  The only thing that uses the number is the actual protocol buffer inside the generated code.
I actually debated on if this should have been added to.